tests/symbols/spec.svg

```diff
@@ -0,0 +1,16 @@
+<svg id="svgElement" width="500" height="500" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should probably add more cases:
 * different transforms (on the `symbol`, on the `use`, "around" the `use`, ...)
 * style inheritance from "around" the `use` element